File: 1291. Sequential Digits/sequentialDigits.cpp
class Solution
{
public:
    vector<int> sequentialDigits(int low, int high)
    {

        vector<int> ans;
        string s = "123456789";
        int n = s.size();
        for (int len = 2; len <= n; len++)
        {
            for (int startInd = 0; startInd <= n - len; startInd++)
            {
                string temp = s.substr(startInd, len);
                int val = stoi(temp);

                if (val >= low && val <= high)
                    ans.push_back(val);
            }
        }
        return ans;
    }
};

File: 1291. Sequential Digits/sequentialDigits.java
class Solution {
    public List<Integer> sequentialDigits(int low, int high) {
        List<Integer> ans = new ArrayList<>();
        String s = "123456789";
        int n = s.length();

        for (int len = 2; len <= n; len++) {
            for (int startInd = 0; startInd <= n - len; startInd++) {
                String temp = s.substring(startInd, startInd + len);
                int val = Integer.parseInt(temp);

                if (val >= low && val <= high) {
                    ans.add(val);
                }
            }
        }

        return ans;
    }
}